src/main/java/com/novocode/junit/EventDispatcher.java

```diff
@@ -151,12 +158,16 @@ void logTo(RichLogger logger) {
   private void postIfFirst(AbstractEvent e)
   {
     e.logTo(logger);
-    if(reported.add(e.fullyQualifiedName())) handler.handle(e);
+    if(reported.add(e.fullyQualifiedName())) {
```

Comment:
I generally would put a space after `if`, `for`, etc. Can we consider changing throughout this PR?

Reply:
These extra spaces always look wrong to me. Most of the code here is probably without spaces (unless someone else rewrote it in the mean time). My style changed over time to put opening braces at the end of the line even in Java and this is inconsistent in the codebase, too. This could be made consistent in a separate PR but honestly I'm not too bothered by it.